src/screens/TermsConditionScreen.js

```diff
@@ -1,74 +1,106 @@
+              clear as possible.
+              {"\n"}
+              This App provides information only and does not operate to as a
+              diagnosis tool and it does not subtitute for professional medical advice
+              and assistance.
+              {"\n"}
+              You will be provided vaccination recommendation and also you can manage
+              your vaccination history through this App. your information such as
+              gender and birth date will not be used anywhere except in this app for
+              recommending vaccination.
+              {"\n"}
+              By clicking below, you acknowledge and agree to our use of your personal
+              information we collect through the use of this app in the ways described
+              in our privacy notice.
+              
+              {"\n"}
+            </Text>
+          </ScrollView>
+        </View>
 
-      <Text style={styles.bodyText}>
-        Date Modified: March 18, 2021 {"\n"} {"\n"}
-        Thank you for using Health Guard! {"\n"}
-        We're happy you're here. Please read this Terms of Service agreement
-        carefully before accessing or using Health Guard. Because it is such an
-        important contract between us and our users, we have tried to make it as
-        clear as possible.
-        {"\n"}
-        This App provides information only and does not operate to as a
-        diagnosis tool and it does not subtitute fot professional medical advice
-        and assistance.
-        {"\n"}
-        You will be provided vaccination recommendation and also you can manage
-        your vaccination history through this App. your information such as
-        gender and birth date will not be used anywhere except in this app for
-        recommending vaccination.
-        {"\n"}
-        By cliking below, you acknowledge and agree to our use of your personal
-        information we collerect through the use of is app in the ways described
-        in our privacy notice.
-        {"\n"}
-      </Text>
-      <Button
-        title=" I agree "
-        onPress={() => navigation.navigate("ChildrenScreen")}
-      />
+
+        <View style={styles.buttonContainer}>
+          <TouchableOpacity
+            style={screenConfig.buttonContainer}
+            onPress={() => navigation.navigate("ChildrenScreen")}
+          >
+            <Text style={screenConfig.buttonText}>LET'S START</Text>
+          </TouchableOpacity>
+        </View>
+
+      </View>   
     </SafeAreaView>
   );
 };
 
 const styles = StyleSheet.create({
-  container: {
-    backgroundColor: colors.white,
+  screenContainer: {
     flex: 1,
+    marginTop: verticalScale(40),
+    paddingLeft: moderateScale(24),
+    paddingRight: moderateScale(24),
+    backgroundColor: colors.back
   },
-  text: {
-    textAlign: "center",
-    fontSize: moderateScale(18),
-    fontWeight: "bold",
+  titleContainer: {
+  },
+  cardContainer: {
+    flex: 1,
+    elevation: 1,
```

Comment:
The size of card looks so small on my view...how about put some exact size (width and height) on it?